### DISCUSSION TITLE
Using Taiko on websites with a captcha

### DISCUSSION THREAD

**Discussion by namelus** [getgauge/taiko#1927] [Development]
Good day all

Anyone have any experience working with websites that have captcha in them?

Thanks1

**Comment by namelus** [comment]
@zabil - Saw that there was an issue pertaining to this query of mine. Any ideas or suggestions on this?

**Reply by zabil** [reply]
Captcha's are designed to stop automated scripts. So there is no way to avoid captchas using Taiko. 

If you are in control of the test environment, configure the site to not have a captcha.
